Misc/NEWS.d/next/Core and Builtins/2023-03-03-23-21-16.gh-issue-102406.XLqYO3.rst

```diff
@@ -0,0 +1 @@
+codecs encoding/decoding errors now get the context information (which operation and which codecs) attached as PEP-678 notes instead of through chaining a new instance of the exception.
```

Comment:
```suggestion
:mod:`codecs` encoding/decoding errors now get the context information (which operation and which codecs) attached as :pep:`678` notes instead of through chaining a new instance of the exception.
```